### ISSUE TITLE
Numpy deprecation warnings

### ISSUE THREAD

**Issue by toaarnio** [spectralpython/spectral#129]
The unsized basic datatypes `np.float` and `np.int` have been deprecated by Numpy. This is causing warnings at `import spectral`. Example:

```
[...] lib/python3.8/site-packages/spectral/config.py:160: DeprecationWarning: `np.int` is a deprecated alias
for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior
and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the
precision. If you wish to review your current use, check the release note link for additional information.
```

**Comment by tboggs** [comment]
Thanks for the update!
